autoarray/plot/abstract_plotters.py

```diff
@@ -182,6 +182,16 @@ def _subplot_custom_plot(self, **kwargs):
                 except AttributeError:
                     self.figures_1d(**{key: True})
 
+            try:
+                self.mat_plot_2d.subplot_index = max(
+                    self.mat_plot_1d.subplot_index, self.mat_plot_2d.subplot_index
+                )
+                self.mat_plot_1d.subplot_index = max(
+                    self.mat_plot_1d.subplot_index, self.mat_plot_2d.subplot_index
+                )

```

Comment:
Are both these values supposed to be the same? If so make it explicit by computing one and assigning the other

```python
self.mat_plot_2d.subplot_index = max(
    self.mat_plot_1d.subplot_index, self.mat_plot_2d.subplot_index
)
self.mat_lot_1d.subplot_index = self.mat_plot_2d.subplot_index